lua/wikis/commons/ThisDay.lua

```diff
@@ -0,0 +1,347 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:ThisDay
```

Comment:
should we rename it so it is in the folder?

Reply:
imho: create with new name (also dropping old entry points) -> remove usage of old one -> nuke old one

Reply:
what hjp said makes sense